README.md

```diff
@@ -25,14 +25,38 @@ curl https://raw.githubusercontent.com/perl-actions/ci-perl-tester-helpers/maste
 
 ## `auto-build-and-test-dist`
 
+Any argument is forwarded to `test-dist` (which forwards it to `prove`)
+
+Examples:
+```
+auto-build-and-test-dist --state save
+auto-build-and-test-dist --jobs $(nproc --all)
+auto-build-and-test-dist --verbose
+```
+
 ## `build-dist`
 
+Builds your distribution.
+
+- `Dist::Zilla`
+
+`build-dist` recognizes it though it favours `Build.PL` / `Makefile.PL` coming from
```

Comment:
I don't think we should prefer `Build.PL/Makefile.PL`.

Reply:
Please see example job: https://github.com/happy-barney/perl-Test-YAFT/actions/runs/5672580805
for problems when `dist.ini` is preferred 

background info: I'm trying to develop reusable workflow for perl modules (mvp - support my repositories)

Reply:
Right. That breaks your workflow, but the above change breaks mine. ;) 

We'll figure it out. One way might be to set an environment variable which will ignore `dist.ini`. Could be done with a command line option, but that would have to mess with the args before passing them on to `prove`. Or maybe we need a different script.

Reply:
what about 0a4657e ?
- defaults to current behaviour
- allows mixing approaches using CI matrix (more or less tests whether `Build.PL` / `Makefile.PL` are up-to-date-ish